helm/values.yaml

```diff
@@ -68,24 +68,20 @@ headlessServiceName: ""
 
 # Kafka configuration
 config:
-  autoCreateTopicsEnable: true
-  compressionType: gzip
+  autoCreateTopicsEnable: false
+  compressionType: producer
```

Comment:
Using other values can cause recompression at broker.

Reply:
@laxmanchekka isn't Kafka broker smart enough to know the client has already compressed?